src/index.html

```diff
@@ -1,13 +1,77 @@
 <!doctype html>
-<html lang="en">
+<html class="page" lang="en">
```

Comment:
no needs to add a class to html
```suggestion
<html lang="en">
```

Reply:
this is BEM I can't use tag selectors, only classes

Reply:
no, you can use tag selectors for these two tags(body, html)

Reply:
you can use tags for normalizing or reseting default styles